src/commands/upgrade.js

```diff
@@ -72,6 +74,27 @@ exports.handler = makeCommand( { checkDocker: false }, async ( { verbose, env }
 		yaml.services.phpfpm.image = phpImage;
 	}
 
+	if ( undefined !== yaml.services.elasticsearch ) {
```

Comment:
@tylercherpak Minor, but for consistency, should this conditional be written in a non-Yoda conditional format? The rest of the similar conditional checks in this file aren't in Yoda conditional so I wanted to point that out.